RadixWallet/Core/FeaturePrelude/AddressView/AddressDetails+View.swift

```diff
@@ -0,0 +1,266 @@
+extension AddressDetails.State {
+	var viewState: AddressDetails.ViewState {
+		.init(
+			address: address,
+			title: title,
+			qrImage: qrImage,
+			showEnlarged: showEnlarged,
+			showShare: showShare
+		)
+	}
+}
+
+// MARK: - AddressDetails.View
+
+public extension AddressDetails {
```

Comment:
New features should ideally use the observable style. You will need to add at least one `WithPerceptionTracking`, sometimes multiple.